app/views/produce_items/index.html.erb

```diff
@@ -1,11 +1,23 @@
-<h1 class="page__title"> <%= I18n.t('produce_item.index.title') %> </h1>
+<div class="content background-color-blue">
+  <h1 class="page__title banner-small m-bottom-m"> <%= I18n.t('produce_item.index.title') %> </h1>
 
-<table class="produce-table">
-    <% @produce_items.each do |produce|  %>
-      <tr>
-        <td class="produce-table__name"><%= I18n.t("produce_items.names.#{produce.name}").capitalize %></td>
-        <td class="produce-table__category"><%= I18n.t("produce_items.categories.#{produce.category}") %></td>
-        <td class="produce-table__link_action"><%= link_to I18n.t('action.show'), produce, class: "link_action produce_link" %></td>
-      </tr>
+  <div class="nav-actions-holder p-sides-m">
+    <h4><%= link_to I18n.t("action.back").upcase, root_path, class: "link_action back" %></h4>
+  </div>
+
+  <div class="card-holder p-top-l p-bottom-l p-sides-s">
+    <% @produce_items.find_each do |produce| %>
+      <div class="card">
+        <div class="card-inner">
+          <h4 class="produce-name"><%= I18n.t("produce_items.names.#{produce.name}") %></h4>
+          <div class="card-inner__month-name">
+            <%= produce.item_seasons.find_each do |item_season| %>
+              <%= formatted_month_list(produce, item_season) %>
+            <% end %>
```

Comment:
**suggestion:** We could simplify the logic by using array methods. For example

```rb
produce.item_seasons.map { formatted_month_name(_1.month_index) }.join(' /')
```

This could be extracted to the helper if you'd like. One thing to keep in mind is that here you are relying on the order in which seasons were added to the `produce`. So `janeiro /fevereiro` and `fevereiro /janeiro` are equally valid outputs depending on the order `item_seasons` were inserted.

I would recommend making this deterministic by:

```rb
produce.item_seasons.order(month_index: :asc).map { formatted_month_name(_1.month_index) }.join(' /')
```

Reply:
changed the helper to do this:
`produce.item_seasons.order(month_index: :asc).map { formatted_month_name(_1.month_index) }.join(' / ')`

and the index to do this:
`          <div class="card-inner__month-name">
            <%= formatted_month_list(produce) %>
          </div>`